package.json

```diff
@@ -29,17 +27,9 @@
   "devDependencies": {
     "@graphprotocol/graph-cli": "https://github.com/graphprotocol/graph-cli.git",
     "@graphprotocol/graph-ts": "^0.18.1",
-    "babel-polyfill": "^6.26.0",
-    "babel-register": "^6.26.0",
     "cli-confirm": "^1.0.1",
-    "dotenv": "^8.2.0",
     "eslint": "^6.8.0",
     "eslint-config-prettier": "^6.11.0",
-    "ganache-cli": "^6.9.1",
-    "mustache": "^4.0.1",
-    "openzeppelin-solidity": "^2.5.0",
-    "truffle": "^5.1.37",
```

Comment:
I think this truffle is required. Otherwise, it breaks the "yarn abi" script in the README.

![image](https://user-images.githubusercontent.com/2352112/91170898-dab7ed00-e6d9-11ea-8dfe-ce495eaf743d.png)

The other libs, I would say they are OK to delete.

Reply:
Good catch! While `truffle` isn't needed, we do need `openzeppelin-solidity`.

Reply:
Should be fixed now.